### PR TITLE
[FIX] website: display TourPointer on configurator screen

### PR DESCRIPTION
**Pull request by robinlej**
Since [3a79803][1] where the CSS / classes of the TourPointer was completely
redone, the pointer doesn't show up anymore in the default tip on the
configurator waiting screen.
This commit redisplays it as it appeared before.

task-3358028

[1]: https://github.com/odoo/odoo/commit/3a79803

### CODE REVIEW

File: addons/website/static/src/components/website_loader/website_loader.xml [hunk 1]
```diff
@@ -9,7 +9,10 @@
         <p class="o_website_loader_tip w-25">
             <t t-if="state.showTips">
                 TIP: Once loaded, follow the
-                <span class="o_tooltip o_tooltip_visible bottom o_animated position-relative"></span>
+                <span class="o_tour_pointer o_bottom position-absolute ms-2"
+                    style="--TourPointer__height: 1em; --TourPointer__width: 1em;">
```
Comment (rdeodoo) [inline addons/website/static/src/components/website_loader/website_loader.xml:13]: Move this "weird" style inside the scss file, we already have something targetting its parent `o_website_loader_tip`.
Also, you removed the `o_tooltip` which was attached to some style, you need to remove that style :+1:

**Review by rdeodoo — CHANGES_REQUESTED** [review]
You forgot the `:` after `[1]` in your markdown link, so it's actually not a markdown link.

**Review by Guillaume-gdi — COMMENTED** [review]
Code looks good thanks!
I just have two questions:
1. I did a try of the configurator with your changes and during the loading the `tour_pointer` is correctly displayed good :+1: but after the loading I don't see the pointer to guide me to build the perfect page in 7 steps (I try with a new DB, I cleared cache,...) Am I missing something or the pointer do not work after the configurator ?
2. Can't you also remove the scss of `o_tooltip` in `website.theme_install.scss` ?

**Comment by robinlej** [comment]
@Guillaume-gdi Point (1) is handled in task-3084175.
(2) is a nice catch I hadn't seen. It even looks like the whole file is obsolete, those classes aren't used anywhere else in the codebase.

**Comment by Guillaume-gdi** [comment]
> the whole file is obsolete

Ok @robinlej if the scss class is not used since 16.3, I propose you remove it in the master forward port of this PR :+1:

**Review by Guillaume-gdi — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**